.jenkins/JenkinsfileArm

```diff
@@ -39,15 +39,45 @@ node {
             sh 'git submodule update --init test third_party/GCutil'
         }
 
-        stage('Prepare build') {
+        withCredentials([usernamePassword(credentialsId: 'arm32docker', passwordVariable: 'passwordVariable', usernameVariable: 'usernameVariable')]) {
+            def remote = [:]
+            remote.name = 'arm32-docker'
+            remote.host = 'localhost'
+            remote.port = 11111
+            remote.user = usernameVariable
+            remote.password = passwordVariable
+            remote.allowAnyHosts = true
+            stage('prepare workspace') {
+                sshCommand remote: remote, command: "rm -rf escargot escargot.zip"
+                sshCommand remote: remote, command: "pwd"
+                sh "zip -r escargot.zip ./*"
+                sshPut remote: remote, from: "escargot.zip", into: "./"
+                sshCommand remote: remote, command: "unzip escargot.zip -d escargot"
+            }
+            stage('Build on arm32') {
+                sshCommand remote: remote, command: "\
+                cd escargot;\
+                cmake -H./ -Bout -DCMAKE_SYSTEM_NAME=Linux -DCMAKE_SYSTEM_PROCESSOR=arm32 -DESCARGOT_MODE=release -DESCARGOT_THREADING=ON -DESCARGOT_TEMPORAL=ON -DESCARGOT_TCO=ON -DESCARGOT_TEST=ON -DESCARGOT_OUTPUT=shell -GNinja;\
+                cmake --build ./out/;\
+                "
+            }
+            stage('Running tests on arm32') {
+                sshCommand remote: remote, command: "\
+                cd escargot;\
+                tools/run-tests.py --engine='/root/escargot/out/escargot' --extra-arg='--skip intl402' new-es v8 spidermonkey chakracore test262;\
+                "
+            }
+        }
+
+        stage('Prepare build on aarch64') {
             sh 'cmake -H./ -Bout -DESCARGOT_MODE=release -DESCARGOT_THREADING=ON -DESCARGOT_TEMPORAL=ON -DESCARGOT_TCO=ON -DESCARGOT_TEST=ON -DESCARGOT_OUTPUT=shell -GNinja'
         }
 
-        stage('Build') {
+        stage('Build on aarch64') {
             sh 'cmake --build ./out/'
         }
 
-        stage('Running test') {
+        stage('Running test on aarch64') {
```

Comment:
It seems like that tests on `arm` and `aarch64` are done in sequential.
But, is it possible to run the both test in parallel?

Reply:
Good point. I can fix it